hello_jsontype.py

```diff
@@ -0,0 +1,57 @@
+from metaflow import FlowSpec, step, Parameter, JSONType
+
+
+class HelloJSONType(FlowSpec):
+    """
+    A flow for testing ability to handle JSONType objects.
+
+    """
+
+    str_to_list = Parameter("str_to_list",
+                            type=JSONType,
+                            default="[1, 2, 3]")
+    str_to_dict = Parameter("str_to_dict",
+                            type=JSONType,
+                            default='{"one": 1, "two": 2, "three": 3}')
+    list_param = Parameter("list_param",
+                           type=JSONType,
+                           default="[1, 2, 3]")
+    dict_param = Parameter("dict_param",
+                           type=JSONType,
+                           default='{"one": 1, "two": 2, "nine": 9}')
+
+    @step
+    def start(self):
+        """
+        This is the 'start' step. All flows must have a step named 'start' that
+        is the first step in the flow.
+
+        """
+        print("HelloJSONType is starting.")
+        self.next(self.hello)
+
+    @step
+    def hello(self):
+        """
+        A step to print out JSONType objects.
+
+        """
+        print("Ready to begin testing JSONType object params")
+        print(f"str_to_list - type: {type(self.str_to_list)}, data: {self.str_to_list}")
+        print(f"str_to_dict - type: {type(self.str_to_dict)}, data: {self.str_to_dict}")
+        print(f"list_param - type: {type(self.list_param)}, data: {self.list_param}")
+        print(f"dict_param - type: {type(self.dict_param)}, data: {self.dict_param}")
```

Comment:
I suggest asserting that all of these parameters have NoneProd data, and actually assert that the values equal to what you pass in from `test_run_jsontype.py`. This way you will know that the test fails if corresponding metaflow run fails.

Reply:
I will do this for JSONType tests and all other tests in [AIP-6471]. I pasted this feedback in a comment on that task.